condor_watch_q.py

```diff
@@ -515,30 +532,38 @@ def table_by_batch_name(self):
                     # the queue
                     continue
                 d = data[batch_name]
+                
                 for proc_status in proc_statuses.values():
                     d[proc_status] += 1
 
                 live_proc_ids = [
                     p for p, status in proc_statuses.items() if status in ACTIVE_STATES
                 ]
-
+  
                 if len(live_proc_ids) > 0:
                     if len(live_proc_ids) == 1:
                         x = "{}.{}".format(cluster_id, live_proc_ids[0])
                     else:
-                        x = "{}.{}-{}".format(
-                            cluster_id, min(live_proc_ids), max(live_proc_ids)
+                        x = "{}.{}".format(
+                              cluster_id, min(live_proc_ids), max(live_proc_ids)
                         )
                     live_job_ids[batch_name].append(x)
 
+	    maxmin_job = collections.defaultdict(list)
+        for id in live_job_ids:
```

Comment:
Could you clarify what the intent is here? This is the multiple-ranges-of-job-ids elision, right?